app/ide-desktop/lib/dashboard/src/layouts/UserBar.tsx

```diff
@@ -68,26 +69,41 @@ export default function UserBar(props: UserBarProps) {
     <FocusArea active={!invisible} direction="horizontal">
       {innerProps => (
         <div
-          className="pointer-events-auto flex h-row shrink-0 cursor-default items-center gap-user-bar rounded-full bg-frame px-icons-x pr-profile-picture backdrop-blur-default"
+          className="pointer-events-auto flex h-row shrink-0 cursor-default items-center gap-1 rounded-full bg-frame px-icons-x pr-profile-picture backdrop-blur-default"
           {...innerProps}
         >
-          <Button
-            active={isHelpChatOpen}
-            image={ChatIcon}
+          <ariaComponents.Button
+            variant="icon"
+            size="custom"
+            className="mr-1"
```

Comment:
normally we use gaps to control spacing in flex and grid containers - is there a reason we meed extra margins here?

Reply:
![CleanShot 2024-05-07 at 12 05 43@2x](https://github.com/enso-org/enso/assets/61194245/6082b1d6-88d0-43d7-9269-332a1e3055f2)

Yeah, extra margin makes sense here because otherwise the gap bw chat and the next button is too narrow.
Gaps are good when we need to put spaces between equal-sized elements. otherwise, it's better to use margins

Reply:
i guess... i think in this situation `p-1` between the icon's container and the icon would be preferable to make the elements the same size, but i don't know whether it'd make the left side look weird

Reply:
Using margins is preferable here.